update_requirements.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+for FUNCTION in example_function1 example_function2
+do
+	poetry export -f requirements.txt --without-hashes > $FUNCTION/requirements.txt
```

Comment:
Adding this line in the README along with some words to describe the workflow to update would be enough, for now.